### PR TITLE
made some coffee with python

### PR DESCRIPTION
**Pull request by TheRakken**

### CODE REVIEW

File: fucking_coffee.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import datetime
+import sys
+import subprocess
+import telnetlib
+import time
+
+today = datetime.date.today()
+
+# skip weekends
+if today.strftime('%A') == 'Saturday' || today('%A') == 'Sunday':
```
Comment (ekohl) [inline fucking_coffee.py]:
```python
if today.strftime('%A') in ('Saturday', 'Sunday'):
```

Also fixes a bug where you forgot stftime on sunday.

Comment (TheRakken) [inline fucking_coffee.py]: Thanks!

File: fucking_coffee.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+con.close()
+
+
+
+
+
+
+
+
+
```
Comment (NARKOZ) [inline fucking_coffee.py]: Remove extra lines :scissors:

**Comment by NARKOZ** [comment]
Please `chmod +x` and place the file under `python` directory.

**Pull request closed without merging**